.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,22 +10,15 @@ about: Create a bug report to help us improve
 _Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_
 
 - [ ] 1. The issue provides a reproduction available on [Github](https://github.com/Urigo/graphql-mesh/tree/master/examples/hello-world), [Stackblitz](https://stackblitz.com/github/Urigo/graphql-mesh/tree/master/examples/hello-world) or [CodeSandbox](https://codesandbox.io/s/github/Urigo/graphql-mesh/tree/master/examples/hello-world)
-
-_Make sure to fork this template and run `yarn generate` in the terminal._
-
-_Please make sure the Codegen and plugins version under `package.json` matches yours._
-
+> Make sure to fork this template and run `yarn generate` in the terminal.
+>
+> Please make sure the Codegen and plugins version under `package.json` matches yours.
```

Comment:
I think we have codegen here by mistake.

Reply:
oops, fixed :)